package.json

```diff
@@ -6,8 +6,12 @@
   "scripts": {
     "test": "grunt karma",
     "validate": "grunt sasslint && grunt jshint",
-    "build": "grunt build",
-    "watch": "grunt watch"
+    "buildJS": "grunt buildJS",
+    "buildCSS": "grunt buildCSS",
+    "build": "grunt buildJS && grunt buildCSS",
+    "sync": "grunt rsync",
+    "watch": "grunt watch",
```

Comment:
Since the `default` grunt task already simply runs `watch`, perhaps we don't need this script?